inc/3rdparty/diff/README.txt

```diff
@@ -0,0 +1,5 @@
+## Versions
```

Comment:
It may be fine to just mention versions in commits or related PRs

Reply:
I thought having the versions explicitly in the source control would help with documenting and upgrading as it is easier to find the current version used.